internal/exec/shell_utils.go

```diff
@@ -1,13 +1,18 @@
 package exec
 
 import (
+	"bytes"
+	"context"
 	"fmt"
 	"os"
 	"os/exec"
 	"runtime"
 	"strings"
 
 	u "github.com/cloudposse/atmos/pkg/utils"
+	"mvdan.cc/sh/v3/expand"
```

Comment:
Why use this syntax instead of 
```suggestion
	"github.com/mvdan/sh/v3/expand"
```

Reply:
It was a simple oversight, suggestion applied !

Reply:
Well actually the module really advertises itself as `mvdan.cc/sh/v3` so I reverted my last commit (was incomplete anyway) based on your suggestion.